File: src/Omnia.ProcessManagement.Models/ProcessTypes/PublishingApprovalSettings/LimitedUsersPublishingApprovalSettings.cs
using Omnia.Fx.Models.Users;
using System;
using System.Collections.Generic;
using System.Text;

namespace Omnia.ProcessManagement.Models.ProcessTypes.PublishingApprovalSettings
{
    public class LimitedUsersPublishingApprovalSettings : PublishingApprovalSettings
    {
        public override PublishingApprovalSettingsTypes Type
        {
            get
            {
                return PublishingApprovalSettingsTypes.LimitedUsers;
            }
        }

        public List<UserIdentity> Users { get; set; }
    }
}

File: src/Omnia.ProcessManagement.Web/client/fx/models/data/processtypes/publishingapprovalsettings/LimitedUsersPublishingApprovalSettings.ts
import { PublishingApprovalSettings, PublishingApprovalSettingsTypes } from '.';
import { UserIdentity } from '@omnia/fx-models';

export interface LimitedUsersPublishingApprovalSettings extends PublishingApprovalSettings {
    type: PublishingApprovalSettingsTypes.LimitedUsers;
    users: Array<UserIdentity>;
}